### PR TITLE
Person details frontend

### PR DESCRIPTION
**Pull request by Kutaiba-Nezar-Kashmar**
- Made front end part
  - components 
  - page
  - services
  - models
- Backend update

### CODE REVIEW

File: src/Web/couch-potatoes-web-app/src/components/person/PersonStatsInformation.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+const PersonStatsInformation: FC<Stats> = ({
+                                               numberOfMovies,
+                                               averageMoviesRatingsAsACast,
+                                               averageMoviesRatingsAsACrew,
+                                               knownForGenre
+                                           }) => {
+    return (
+        <>
+            <Heading color="white" as='h4' size='md'>Stats</Heading>
+            <br/>
```
Comment (Kasper-S-Jensen) [inline src/Web/couch-potatoes-web-app/src/components/person/PersonStatsInformation.tsx:22]: why you break

**Review by michaelbui99 — APPROVED** [review]
LGTM, HE DONT MISS 🔥🔥🔥🔥🔥🔥🔥🔥

**Review by Kasper-S-Jensen — APPROVED** [review]
LGTM 😘

File: src/Web/couch-potatoes-web-app/src/components/person/PersonStatsInformation.tsx (hunk 1 above)
Comment (michaelbui99) [reply src/Web/couch-potatoes-web-app/src/components/person/PersonStatsInformation.tsx:22]: 🔥🔥🔥 catch!!!

**Pull request merged**